packages/governance/test/unitTests/test-buildParamManager.js

```diff
@@ -213,7 +213,7 @@ test('Invitation', async t => {
   t.is(paramManager.getAmount('Amt'), drachmaAmount);
   const invitationActualAmount =
     paramManager.getInvitationAmount('Invite').value;
-  t.is(invitationActualAmount, invitationAmount.value);
+  t.deepEqual(invitationActualAmount, invitationAmount.value);
```

Comment:
Huh, why did this need to change?

Reply:
Surprising to me too. Running the test without this change,
```
  Values are deeply equal to each other, but they are not the same:

  [
    {
      description: 'simple',
      handle: Object @Alleged: InvitationHandle {},
      installation: Object @Alleged: Installation {
        getBundle: Function getBundle {},
      },
      instance: Object @Alleged: InstanceHandle {},
    },
  ]

  › packages/governance/test/unitTests/test-buildParamManager.js:216:5
```
Comparing each named key, the are the same value,
```
  for (const k of Object.keys(invitationActualAmount[0])) {
    t.is(invitationActualAmount[0][k], invitationAmount.value[0][k]);
  }
```
But the object containing them is different. Do you know why that is @erights ?

Reply:
> But the object containing them is different. Do you know why that is @erights ?

Yes. The amount is a copyRecord, which is pass-by-copy. It is serialized into persistent storage, and restored from persistent storage by unserializing. This preserves `keyEQ` equivalence which happens to be similar enough to AVA's `deepEqual` equivalence.

Reply:
I added a `keyEQ` test to clarify.

Reply:
Got it. Tests also serve as documentation of how things are expected to work, so would it be more accurate to use `keyEQ` here?

```suggestion
  t.true(keyEQ(invitationActualAmount, invitationAmount.value));
```